### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by hultenvp** [hultenvp/solis-sensor#376] [Announcements]
It's getting increasingly difficult for me to create time to spend on this integration.
To ensure some degree of continuity it would be great if there are one or two of you who are willing to join forces on this project.
Having coding experience, preferably python, is important as it will be difficult for me to spend significant time on reviews as well.
It's not just maintaining. One of my biggest plans is to still bring the integration to the next level by publishing [Soliscloud API](https://github.com/hultenvp/soliscloud_api) on PyPI and writing a new integration that automatically supports all sensors the inverter supports.
So if you are interested then drop a line in the comments or reach out to me on Discord (petervanhulten).

**Comment by jmason** [comment]
Absolutely -- I've worked on plenty of OSS projects before and would be happy to help (although I'm not 100% on what specific requirements HomeAssistant have for their integrations).  also my Python's not superb -- but workable :)

**Reply by hultenvp** [reply]
Hi @jmason, that's much appreciated.

I'm also still learning as I go, so no worries. Would be great if we can have a chat via Discord or whatever works for you.

Cheers,
Peter

**Reply by jmason** [reply]
sure -- I've sent a friend request from my discord account (jmason5784).

**Comment by patduckless** [comment]
I would be willing to throw my hat in the ring and help out with this. I find the project quite useful. I'm farily new to using python but can get things to run. I've added you on discord.

**Reply by hultenvp** [reply]
Thanks, missed this message. Appreciated
